<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Immersion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-size: 14pt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,999">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Source Lang</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sourceLang">
          <property name="text">
           <string>Danish</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Target Lang</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="targetLang">
          <property name="text">
           <string>English</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="inputText">
        <property name="styleSheet">
         <string notr="true">font-size: 14pt;</string>
        </property>
        <property name="plainText">
         <string>Hej jeg heder Alex. Hvad hedder du?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="goButton">
        <property name="text">
         <string>GO</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset_OpenAI_API_key"/>
    <addaction name="actionOpen_corrections_folder"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionReset_OpenAI_API_key">
   <property name="text">
    <string>Reset OpenAI API key</string>
   </property>
  </action>
  <action name="actionOpen_corrections_folder">
   <property name="text">
    <string>Open corrections folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
